test/avt/detect-ws.xspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<x:description
+	xmlns:mirror="x-urn:test:mirror"
+	xmlns:myv="http://example.org/ns/my/variable" 
+	xmlns:x="http://www.jenitennison.com/xslt/xspec">
+
+	<x:scenario label="Detect if reformatting interfered with whitespace characters that are important">
+		<!-- Use unparsed-text instead of doc() to avoid having
+			attribute value normalization convert the tabs and
+			line feeds to spaces. -->
+		<x:call function="mirror:param-mirror">
+			<x:param select="$x:xspec-uri => unparsed-text()" />
```

Comment:
Removing the call to `mirror:param-mirror` should still work:

```suggestion
		<x:call function="unparsed-text">
			<x:param select="$x:xspec-uri" />
```

And then `xmlns:mirror="x-urn:test:mirror"` in this file and `<x:helper stylesheet="mirror.xsl"/>` in `avt-ws_schematron.xspec` can be removed.

Reply:
Oops. Disregard my comment. `external_avt-ws_stylesheet.xspec` doesn't work with `<x:call function="unparsed-text">`.